spec/services/republish_manual_service_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "fast_spec_helper"
+require "republish_manual_service"
+
+RSpec.describe RepublishManualService do
+  let(:manual_id) { double(:manual_id) }
+  let(:manual_repository) { double(:manual_repository) }
+  let(:listener) { double(:listener) }
+  let(:listeners) { [listener] }
+  let(:manual) { double(:manual, id: manual_id, slug: "abc") }
+  let(:tag_fetcher) { double(:tag_fetcher) }
+
+  subject {
+    RepublishManualService.new(
+      manual_id: manual_id,
+      manual_repository: manual_repository,
+      listeners: listeners,
+    )
+  }
+
+  before do
+    allow(manual_repository).to receive(:fetch) { manual }
+    allow(manual).to receive(:published?).and_return(true)
+    allow(manual).to receive(:update)
+    allow(listener).to receive(:call)
+    allow(tag_fetcher).to receive(:tags).and_return(
+      [
+        OpenStruct.new(
+          details: OpenStruct.new(
+            type: "specialist_sector",
+          ),
+          slug: "government-digital-guidance/content-publishing",
+        )
+      ]
+    )
+    allow(TagFetcher).to receive(:new).and_return(tag_fetcher)
+  end
+
+  it "publishes the manual" do
```

Comment:
Wouldn't it be `republishes` the manual? It might be an idea for this test to also check that a draft Manual doesn't get published.